src/wallet/rpcwallet.cpp

```diff
@@ -163,6 +218,61 @@ static void WalletTxToJSON(const CWalletTx& wtx, UniValue& entry)
         entry.pushKV(item.first, item.second);
 }
 
+static void WalletTxToCsv(std::vector<std::string>& csvRecord, const CWalletTx& wtx)
+{
+    int confirms = wtx.GetDepthInMainChain();
+    csvRecord[TRANSACTION_CSV_FIELD_CONFIRMATION] = std::to_string(confirms);
+    if (wtx.IsCoinBase())
+        csvRecord[TRANSACTION_CSV_FIELD_GENERATED] = "true";
+    if (confirms > 0)
+    {
+        csvRecord[TRANSACTION_CSV_FIELD_BLOCKHASH] = wtx.hashBlock.GetHex();
+        csvRecord[TRANSACTION_CSV_FIELD_BLOCKINDEX] = std::to_string(wtx.nIndex);
+        csvRecord[TRANSACTION_CSV_FIELD_BLOCKTIME] = std::to_string(LookupBlockIndex(wtx.hashBlock)->GetBlockTime());
+    }
+    else
+    {
+        csvRecord[TRANSACTION_CSV_FIELD_TRUSTED] = wtx.IsTrusted() ? "true" : "false";
+    }
+
+    uint256 hash = wtx.GetHash();
+    csvRecord[TRANSACTION_CSV_FIELD_TXID] = hash.GetHex();
+
+    std::string conflicts = "";
+    for (const uint256& conflict : wtx.GetConflicts())
+    {
+        conflicts += conflict.GetHex();
+        conflicts += ",";
+    }
+    csvRecord[TRANSACTION_CSV_FIELD_WALLETCONFLICTS] = conflicts;
+    csvRecord[TRANSACTION_CSV_FIELD_TIME] = std::to_string(wtx.GetTxTime());
+    csvRecord[TRANSACTION_CSV_FIELD_DATETIME_HUMAN_READABLE] = FormatISO8601DateTime(wtx.GetTxTime());
+    csvRecord[TRANSACTION_CSV_FIELD_TIMERECEIVED] = std::to_string(wtx.nTimeReceived);
+    csvRecord[TRANSACTION_CSV_FIELD_COMPUTETIME] = std::to_string(wtx.nComputeTime);
+
+    // Add opt-in RBF status
+    std::string rbfStatus = "no";
+    if (confirms <= 0) {
+        LOCK(mempool.cs);
+        RBFTransactionState rbfState = IsRBFOptIn(*wtx.tx, mempool);
+        if (rbfState == RBFTransactionState::UNKNOWN)
+            rbfStatus = "unknown";
+        else if (rbfState == RBFTransactionState::REPLACEABLE_BIP125)
+            rbfStatus = "yes";
+    }
+    csvRecord[TRANSACTION_CSV_FIELD_BIP125_REPLACEABLE] = rbfStatus;
+
+    /*
```

Comment:
Finish the todo or delete the commented code.

Reply:
Are you waiting on me on anything here?